core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -855,6 +856,65 @@ default HttpResponse peekHeaders(Consumer<? super ResponseHeaders> action) {
         return of(stream);
     }
 
+    /**
+     * Applies the specified {@link Consumer} to the {@link HttpData}s
+     * emitted by this {@link HttpResponse}.
+     *
+     * <p>For example:<pre>{@code
+     * HttpResponse response = HttpResponse.of("data1,data2");
+     * HttpResponse result = response.peekData(data -> {
```

Comment:
nit: peeked?
```suggestion
     * HttpResponse peeked = response.peekData(data -> {
```
because `result` is somewhat vague in this context?

Reply:
I also think `peeked` looks better so fixed it!